### PR TITLE
Login mobile

### PR DESCRIPTION
**Pull request by TouringIdk**
There was a small bug where the mobile menu appeared in the login and register section, that problem has been overridden making it disappear

### CODE REVIEW

File: src/components/home/Home.js [hunk 1]
```diff
@@ -12,38 +14,46 @@ import Car from './Car';
...
+        authorization: localStorage.getItem('token'),
       },
-    }).then((data) => {
-      data.json().then((dataJson) => dispatch(models(dataJson)));
-    });
+    })
+      .then((response) => response)
+      .then((response) => {
+        const { data } = response;
+        console.log(data);
```
Comment (m05e5) [inline src/components/home/Home.js]: Thanks for your work. I just suggest if you can try removing the "console.logs" in the code to make it look more professional

**Review by m05e5 — APPROVED** [review]

File: src/components/home/Home.js (hunk 1 above)
Comment (TouringIdk) [reply src/components/home/Home.js]: Sure, i am on it

**Comment by TouringIdk** [comment]
Ready!

**Comment by m05e5** [comment]
Thanks for the changes you implemented 👍 . We can merge it.

**Pull request merged**